poly-commitment/src/commitment.rs

```diff
@@ -435,7 +435,7 @@ pub fn combined_inner_product<F: PrimeField>(
 
         if let Some(m) = shifted {
             // polyscale^i sum_j evalscale^j elm_j^{N - m} f(elm_j)
-            let last_evals = if *m > evals.len() * srs_length {
+            let last_evals = if *m >= evals.len() * srs_length {
```

Comment:
I really think we ought to move to this simplified function: https://github.com/o1-labs/proof-systems/pull/448/files and handle chunked polynomials outside of the API

Reply:
btw the reason I couldn't move to that code is the empty polynomials (like selectors when you don't use them)

Reply:
Perhaps, but I don't think we should block this PR on it.